content/300-guides/050-database/880-troubleshooting-orm/100-help-articles/425-nextjs-prisma-client-monorepo.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaTitle: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaDescription: 'Using Prisma Client in a Next.js project in a monorepo setup'
+---
+
+## Problem
+
+If you use Prisma Client in a [Next.js](https://nextjs.org/) application within a monorepo, you will likely run into an error such as the following:
+
+```
+Error: ENOENT: no such file or directory, open '/prisma-next-mono-reproduction/packages/service/.next/cache/webpack/client-development/schema.prisma'
```

Comment:
This error would not actually appear any more in recent versions, where we ahve a custom error message, right @millsp? We should probably mention that error message as an alterantive failure mode here (I think it links to this page) so people understand this is about _their_ problem they just had.

Reply:
Yes, that's correct, the error message has been changed to point the users to this documentation page. So this would be the old message.

Reply:
I think this should be expanded with the new error message as we spoke about in Slack @sabinadams